docs/security.html

```diff
@@ -1087,13 +1087,21 @@ <h4><a id="security_authz_cli" href="#security_authz_cli">Command Line Interface
             <td>Resource</td>
         </tr>
         <tr>
-            <td>--resource-name-type [name-type]</td>
-            <td>Specifies the resource name type to use.<br>
+            <td>--resource-pattern-type [pattern-type]</td>
+            <td>Specifies the resource pattern type to use.<br>
                 Valid values are:<br>
                 <ul>
-                    <li><b>Literal</b> Match resource names exactly or, in the case of the Wildcard name '*', match all resources.</li>
-                    <li><b>Prefixed</b> Match any resource whose name starts with the prefix.</li>
-                    <li><b>All</b> (list|remove only) Matching any name type, including the Wildcard name.</li>
+                    <li><b>Literal (default):</b> a pattern that only matches a resource if the name is an exact match, or, in the case of the Wildcard name '*', matches all resources.
+                        All operations, (add, remove, list), will only affect or return acls on literal resource patterns with the exact supplied name.
```

Comment:
Is this intended?

Reply:
Erm... yes...?

Happy to replace if you can word it better :D